src/trackers/TVC.py

```
@@ -139,106 +176,141 @@ async def check_image_hosts(self, meta):
-        resolution_id = await self.get_res_id(meta['tv_pack'] if 'tv_pack' in meta else 0, meta['resolution'])
-        # this is a different function that common function
-        await self.unit3d_edit_desc(meta, self.tracker, self.signature, image_list, approved_image_hosts=self.approved_image_hosts)
-
-        if meta['anon'] == 0 and not self.config['TRACKERS'][self.tracker].get('anon', False):
-            anon = 0
-        else:
-            anon = 1
+
+        # load MediaInfo.json
+        try:
+            content = await self.read_file(f"{meta['base_dir']}/tmp/{meta['uuid']}/MediaInfo.json")
```

Ruff (0.14.8)

Probable insecure usage of temporary file or directory: "/tmp/"

(S108)